galactic/distribution.yaml

```diff
@@ -2818,13 +2818,10 @@ repositories:
       url: https://gitlab.com/ApexAI/performance_test.git
       version: 1.0.0
```

Comment:
As mentioned in https://github.com/ros/rosdistro/pull/33763#discussion_r916281228 it makes sense to use a branch for doc updates rather than a tag.